Projects/DibyDot/.swiftlint.yml

```
@@ -0,0 +1,120 @@
+
+file_length:
+  warning: 400
+  error: 1000
+
+function_body_length:
+  warning: 40
+  error: 100
+
+type_body_length:
+  warning: 300
+  error: 500
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)